.github/workflows/release.yaml

```diff
@@ -0,0 +1,36 @@
+---
+
+name: Release
```

Comment:
Maybe in the near future we could include in the same workflow the release of the images as well. since this is just the building the binary